kemeny_sourcemash.tex

```diff
@@ -59,15 +59,41 @@ \subsection{Development: localhost}
 
 \section{Technical Description}
 
-While 
+Sourcemash was built to be used as an RSS feed reader, but at its core is a categorization algorithm. We will break down the technical description into these two components: the categorization algorithm and the RSS reader that incorporates the algorithm to allow users to get parse through the news faster.
 
-\subsection{Subsection 1}
+\subsection{Categorization Algorithm}
 
-When
+Because we need articles to group together by category, it was import that our categorization algorithm generate categories with a decent number of overlapping categories. In addition, it was important that the algorithm work quickly because it would need to process many articles 24/7 to stay up-to-date with the latest news.
```

Comment:
The categorization engineer has two fundamental implementation requirements. First, because Sourcemash aims to group or cluster articles by category, it is critical that the categoriztion algorithm generates categories with plentiful overlapping categories among articles. Second, the algorithm must work quickly and efficiently, as it needs to continuously process articles in order to stay up-to-date with the latest news.